Drivers/rc_receiver/Inc/sbus_receiver.hpp

```diff
@@ -37,6 +37,8 @@ class SBUSReceiver{
         public variable
     */
         uint8_t raw_sbus_[SBUS_FRAME_SIZE];
+        bool getIsDataNew();
+        void setIsDataNew(bool newVal);
```

Comment:
I don't think this set method is required. Just update the variable in the `parse()` method, and whenever the data is read

Reply:
Agree, the isDataNew bool should not be able to be set by SM or anything besides sbus driver.